src/calculateRentalCost.js

```diff
@@ -4,7 +4,12 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  } else if (days < 7) {
+    return days * 40 - 20;
+  } else {
```

Comment:
You can remove the `else` block here for cleaner code, as the `if` statement above contains a `return` and will exit the function if the condition is true.